packages/cdktf-cli/bin/cmds/convert.ts

```diff
@@ -37,8 +37,9 @@ function findFileAboveCwd(
     return fullPath;
   }
 
-  if (fs.existsSync(path.resolve(rootPath, ".."))) {
-    return findFileAboveCwd(file, path.resolve(rootPath, ".."));
+  const parentDir = path.resolve(rootPath, "..");
+  if (fs.existsSync(parentDir) && parentDir !== rootPath) {
```

Comment:
I think a parentDir !== rootPath => we reached the top comment would be nice here

Reply:
Damn, too late 😅

Reply:
But you're right

Reply:
The commit message contains that info, however 😁

Reply:
Should I still follow-up?

Reply:
Nah, I think the commit message should be enough